### PR TITLE
Update patch_wagtailadmin.py

### PR DESCRIPTION
**Pull request by easherma**
make compatible with Wagtail 2.7

### CODE REVIEW

File: wagtail_modeltranslation/patch_wagtailadmin.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def _patch_other_models(self, model):
             panels = filter(lambda field: field.field_name not in translation_registered_fields, panels)
             edit_handler = ObjectList(panels)
 
-            SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to_model(model)
+            SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to(model=model)
```
Comment (DiogoMarques29) [inline wagtail_modeltranslation/patch_wagtailadmin.py]: Add a version check to keep compatibility for wagtail < 2.5

**Comment by DiogoMarques29** [comment]
Hi @easherma 
In order to keep compatibility, can you add a wagtail version check so people using wagtail prior to 2.5 can still use this package?
Cheers

File: wagtail_modeltranslation/patch_wagtailadmin.py (hunk 1 above)
Comment (easherma) [reply wagtail_modeltranslation/patch_wagtailadmin.py]: @DiogoMarques29 I'm not familar with how to do that in this context, could you point me in the right direction?

Comment (DiogoMarques29) [reply wagtail_modeltranslation/patch_wagtailadmin.py]: you could do something like:
```
from wagtail import VERSION
...
if VERSION < (2,5):
    SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to_model(model)
else:
    SNIPPET_EDIT_HANDLERS[model] = edit_handler.bind_to(model=model)
```

**Comment by easherma** [comment]
Closed in favor of https://github.com/infoportugal/wagtail-modeltranslation/pull/258

**Pull request closed without merging**